src/views/Join.vue

```diff
@@ -46,9 +48,12 @@
     </div>
 
     <div class = "cta">
-        <h1> Ready to jump in?  Apply today. </h1>
+        <h1> Not ready to join yet? Sign up for our mailing list!</h1>
```

Comment:
The phrasing "Not ready" suggests that being on the mailing list and joining are mutually exclusive. Ideally, I would use phrasing that would encourage anyone remotely interested with Stanford CTC should join the mailing list. For example, what about "Want to stay in the loop? Sign up for our mailing list!"